<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="4" skipped="0" tests="18" time="83.264" timestamp="2025-08-30T10:19:59.076120" hostname="dut7358"><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesCPU" name="test_all_gather_object_cpu" time="4.112"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2197154">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2197154</failure></testcase><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesCPU" name="test_broadcast_object_list_cpu" time="4.612" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesCPU" name="test_gather_object_cpu" time="4.912" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesCPU" name="test_scatter_object_list_cpu" time="4.811" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesCPU" name="test_send_recv_object_list_cpu" time="4.311" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesCPU" name="test_subpg_all_gather_object_cpu" time="3.810"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2198654">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2198654</failure></testcase><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesCPU" name="test_subpg_broadcast_object_cpu" time="4.611" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesCPU" name="test_subpg_gather_object_cpu" time="4.611" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesCPU" name="test_subpg_scatter_object_cpu" time="4.610" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesXPU" name="test_all_gather_object_xpu" time="4.211"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2199876">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2199876</failure></testcase><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesXPU" name="test_broadcast_object_list_xpu" time="5.013" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesXPU" name="test_gather_object_xpu" time="4.812" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesXPU" name="test_scatter_object_list_xpu" time="4.511" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesXPU" name="test_send_recv_object_list_xpu" time="4.211" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesXPU" name="test_subpg_all_gather_object_xpu" time="3.811"><failure message="AssertionError: Scalars are not equal!&#10;&#10;Expected 0 but got -6.&#10;Absolute difference: 6&#10;Relative difference: inf&#10;Expected exit code 0 but got -6 for pid: 2201375">Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 716, in wrapper
    self._join_processes(fn)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 980, in _join_processes
    self._check_return_codes(fn, elapsed_time)
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_distributed.py", line 1056, in _check_return_codes
    self.assertEqual(
  File "/tmp/xpu-tool/Python/3.10.18/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 4181, in assertEqual
    raise error_metas.pop()[0].to_error(  # type: ignore[index]
AssertionError: Scalars are not equal!

Expected 0 but got -6.
Absolute difference: 6
Relative difference: inf
Expected exit code 0 but got -6 for pid: 2201375</failure></testcase><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesXPU" name="test_subpg_broadcast_object_xpu" time="4.712" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesXPU" name="test_subpg_gather_object_xpu" time="4.711" /><testcase classname="test.distributed.test_c10d_object_collectives.TestObjectCollectivesXPU" name="test_subpg_scatter_object_xpu" time="4.412" /></testsuite></testsuites>